data/eval-machines.nix

```diff
@@ -7,6 +7,7 @@ let
   lib = network.network.lib or nwPkgs.lib or (import <nixpkgs/lib>);
   evalConfig =
     network.network.evalConfig or ((nwPkgs.path or <nixpkgs>) + "/nixos/lib/eval-config.nix");
+  specialArgs = network.specialArgs or { };
```

Comment:
Tried to test this, and it revealed this should be `network.network.specialArgs`

here's my test:

```
nix-shell-env ❯ git diff
diff --git a/examples/simple.nix b/examples/simple.nix
index 7d82623..23f4a6d 100644
--- a/examples/simple.nix
+++ b/examples/simple.nix
@@ -4,6 +4,7 @@ in
 {
   network = {
     inherit pkgs;
+    specialArgs.thing = true;
     description = "simple hosts";
     ordering = {
       tags = [
@@ -13,10 +14,10 @@ in
     };
   };

-  "web01" = _: {
+  "web01" = {thing, ...}: {
     deployment.tags = [ "web" ];

-    boot.loader.systemd-boot.enable = true;
+    boot.loader.systemd-boot.enable = thing;
     boot.loader.efi.canTouchEfiVariables = true;

     services.nginx.enable = true;
```

Reply:
fixed